clients/clients.yaml

```diff
@@ -0,0 +1,16 @@
+ndoli:
+  tests:
+    git-annex-home: |
+      set -ex
```

Comment:
may be even 
```suggestion
      set -exu
```
since variables are used

Reply:
Done.